R/dataframe-frame.R

```diff
@@ -1888,3 +1888,65 @@ dataframe__write_parquet <- function(
     invisible(self)
   })
 }
+
+#' Write to comma-separated values (CSV) file
+#'
+#' @inheritParams lazyframe__sink_csv
+#' @param file File path to which the result will be written.
+#'
+#' @inherit dataframe__write_parquet return
+#' @examples
+#' tmpf <- tempfile()
+#' as_polars_df(mtcars)$write_csv(tmpf)
+#' pl$read_csv(tmpf)
+#'
+#' as_polars_df(mtcars)$write_csv(tmpf, separator = "|")
+#' pl$read_csv(tmpf, separator = "|")
+dataframe__write_csv <- function(
+  file,
```

Comment:
In py-polars, they have the option of this being `None` to return the CSV content as a string, but I'm not sure how to do this here (or this is actually useful)